src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { User } from './User';
+
+export function Todo({ title, completed, user }) {
+  return (
+    <>
+      <h2 className="todo-list__title">{title}</h2>
+      <User {...user} />
+      <p className="todo-list__status">{`${completed}`}</p>
+    </>
+  );
+}
+
+Todo.propTypes = {
+  title: PropTypes.string.isRequired,
+  completed: PropTypes.bool.isRequired,
+  // eslint-disable-next-line
+  user: PropTypes.object.isRequired,
```

Comment:
Should work like this without disabling linter:
```suggestion
  user: PropTypes.shape({}).isRequired,
```

Reply:
Ok, I will keep this in mind